### PR TITLE
Rewards panel

### PR DESCRIPTION
**Pull request by NejcZdovc**

### CODE REVIEW

**Comment by SergeyZhukovsky** [comment]
What is windowId? Do you have a corresponding brave-core commit?

**Comment by NejcZdovc** [comment]
@SergeyZhukovsky https://github.com/brave/brave-core/pull/459

File: include/bat/ledger/publisher_info.h [hunk 1]
```diff
@@ -52,8 +52,8 @@ LEDGER_EXPORT struct PublisherInfoFilter {
   PUBLISHER_MONTH month;
   int year;
   PUBLISHER_EXCLUDE excluded;
-
   std::vector<std::pair<std::string, bool>> order_by;
+  unsigned int min_duration;
```
Comment (ryanml) [inline include/bat/ledger/publisher_info.h]: Any reason this isn't `uint64_t`?

Comment (NejcZdovc) [reply include/bat/ledger/publisher_info.h]: so this is min duration for visit time, which means that max value is 60 (1minute).

**Pull request merged**